repo2docker/buildpacks/base.py

```diff
@@ -118,6 +118,14 @@
 {% endfor %}
 {% endif -%}
 
+# Run launch script
+{% if launch_scripts -%}
+{% for s in launch_scripts -%}
+RUN chmod +x {{ s }}
+ENTRYPOINT ./{{ s }}
+{% endfor %}
+{% endif -%}
+
```

Comment:
question for @yuvipanda, in https://github.com/jupyter/repo2docker/issues/196#issue-288831988, you mentioned that this script should be time limited:

>Not run for more than x seconds.

Do you have an idea of how to do this? Perhaps prefacing this command with the `timeout` command?